data/src/test/java/com/linkedin/data/schema/resolver/TestDataSchemaResolver.java

```diff
@@ -818,19 +819,19 @@ public void testClasspathResourceDataSchemaResolver()
     final ClasspathResourceDataSchemaResolver resolver = new ClasspathResourceDataSchemaResolver();
     final PegasusSchemaParser parser = new SchemaParser(resolver);
 
-    final String existingSchemaName = "com.linkedin.data.schema.ValidationDemo";
-    final String existingPdlSchemaName = "com.linkedin.restli.example.Album";
+    final List<String> existingSchemas = new ArrayList<>();
+    Collections.addAll(existingSchemas, "com.linkedin.data.schema.ValidationDemo",
+        "com.linkedin.restli.example.Album",
```

Comment:
I assume that you purposely choose a different namespace in the test `com.linkedin.rest.li.example` (vs. `com.linkedin.data.schema`) to well test class loader from resource.